charts/apisix-ingress-controller/templates/servicemonitor.yaml

```diff
@@ -30,8 +30,9 @@ metadata:
   {{- end }}
 spec:
   endpoints:
-  - targetPort: http
+  - targetPort: prometheus
     scheme: http
+    path: /apisix/prometheus/metrics
```

Comment:
Do we need to add a new value `serviceMonitor`, like https://github.com/apache/apisix-helm-chart/blob/f14f58dc7fbe8a963b346ab322a93c662f191b3a/charts/apisix/values.yaml#L571

Reply:
I can make the value configurable, and fix it in both places.

Reply:
@zll600 I added `serviceMonitor.path` as configurable option, please review

Reply:
LGTM, Thank you